README.md

```diff
@@ -401,17 +386,73 @@ fixed it needs to be a expression function to get into the proof context:
 ```Ada
 with System;
 
-package body Componolit.Gneiss.Log is
+package body Gneiss.Log is
 
    use type System.Address;
 
    function Initialized (C : Client_Session) return Boolean is
       (C.Label /= System.Null_Address);
 
-end Componolit.Gneiss.Log;
+end Gneiss.Log;
 ```
 
 The initialization checks if `Label` is a valid address. This ensures that all
 procedures that have an `Initialized` precondition can safely use the label.
 It also makes sure that if `malloc` fails in C the session will not be
 initialized.
+
+##  Buildsystem
+
+Gneiss aims to integrate into the existing build systems of the supported platforms.
+On Genode Gneiss components can be built with the native build system. On Linux there
+is no build system that allows defining and building systems.
+
+### Cement
+
+The Cement build system allows designing and building Gneiss systems that run on Linux either
+in a GNU userspace or directly on the kernel. A system consists of a core component that
```

Comment:
"on the kernel" -> as init process

Reply:
Fixed in https://github.com/Componolit/gneiss/pull/173/commits/a4d2d52c614df05fa33fe6b0f4bc1e6482eb2145.